### PR TITLE
Xml data added

### PR DESCRIPTION
**Pull request by hizmailovich**

### CODE REVIEW

File: src/main/java/com/solvd/movie/kafka/parser/XmlParser.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.solvd.movie.kafka.parser;
+
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XMLDocument;
+import lombok.SneakyThrows;
+
+import java.util.Objects;
+
+public class XmlParser {
```
Comment (h1alexbel) [inline src/main/java/com/solvd/movie/kafka/parser/XmlParser.java:9]: this is duplicate of https://github.com/hizmailovich/ReviewMicroservice/pull/10/files/74ecfdc83fd41780382a3456208580f5ee094d5e#diff-d6274374791ec83b8e9f4801dd751f779edeed900b71b9a6962bd8a47d201a9a

File: src/main/resources/producer.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<producer>
+    <bootstrapServers>localhost:9092</bootstrapServers>
```
Comment (h1alexbel) [inline src/main/resources/producer.xml:2]: host is hardcoded

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/java/com/solvd/movie/kafka/parser/XmlParser.java (hunk 1 above)
Comment (hizmailovich) [reply src/main/java/com/solvd/movie/kafka/parser/XmlParser.java:9]: Fixed, I made this class abstract

File: src/main/resources/producer.xml (hunk 2 above)
Comment (hizmailovich) [reply src/main/resources/producer.xml]: Fixed

File: src/main/java/com/solvd/movie/kafka/parser/XmlParser.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.solvd.movie.kafka.parser;
+
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XMLDocument;
+import lombok.SneakyThrows;
+
+import java.util.Objects;
+
+public abstract class XmlParser {
```
Comment (h1alexbel) [inline src/main/java/com/solvd/movie/kafka/parser/XmlParser.java:9]: utility class = final class + private ctor

In your case, I can still construct your ```XmlParser``` using simple inheritance

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/java/com/solvd/movie/kafka/parser/XmlParser.java (hunk 3 above)
Comment (hizmailovich) [reply src/main/java/com/solvd/movie/kafka/parser/XmlParser.java]: Fixed

File: src/main/java/com/solvd/movie/kafka/parser/XmlParser.java (hunk 1 above)
Comment (h1alexbel) [reply src/main/java/com/solvd/movie/kafka/parser/XmlParser.java:11]: its still a copy of that class

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**